src/styles/components/_Logo.scss

```diff
@@ -0,0 +1,61 @@
+.Logo {
```

Comment:
Just wanted to make a note here about our discussion. We have decided not to prefix classes as this is an internal tool and unlikely to conflict with other styles.

Perhaps we should just rename `.logo` to `.hm-logo` incase there are other logos for the project